content/agent/faq/supported-os-versions.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Datadog Agent supported OS versions
+kind: faq
+disable_toc: true
+further_reading:
+- link: "/agent/"
+  tag: "Documentation"
+  text: Learn more about the Datadog Agent
+---
+
+### Agent 5+
+
+|OS| Supported versions|
+|:----|:----|
+|Debian x86_64 | Debian 7 (wheezy) and above (we do not support SysVinit)|
+|Ubuntu x86_64 | Ubuntu 12.04 and above|
+|RedHat/CentOS x86_64| RedHat/CentOS 6 and above |
+|SUSE Enterprise Linux x86_64 | SUSE 11 SP4 and above (we do not support SysVinit)| 
+|Fedora| Fedora 25 and above |
```

Comment:
Not sure what our official support policy should be for Fedora... That said the RHEL/CentOS RPMs we provide _should_ work on Fedora (since Fedora is the upstream for RHEL/CentOS).

Let's start with Fedora 26 though, since 25 is EOL'ed. And let's mention `x86_64` for Fedora too.

Reply:
Fedora revs pretty quickly and is much more likely to introduce a breaking change than RHEL/CentOS. It also EOL's much more rapidly. Given it's current agent install base, do we want to move it to best effort?

Reply:
+1 to moving it to best support

Reply:
👍